test/clj/rems/api/test_applications_v2.clj

```diff
@@ -487,11 +487,11 @@
                                           :event/actor "member"
                                           :application/id 1
                                           :invitation/token token})
-                            expected-application (-> (deep-merge expected-application
-                                                                 {:application/last-activity (DateTime. 5000)
-                                                                  :application/events events
-                                                                  :application/members #{{:userid "member"}}})
-                                                     (assoc :application/invitation-tokens {}))]
+                            expected-application (deep-merge expected-application
+                                                             {:application/last-activity (DateTime. 5000)
+                                                              :application/events events
+                                                              :application/members #{{:userid "member"}}
+                                                              :application/invitation-tokens ^:replace {}})]
```

Comment:
I'd just `merge` the other keys and `update` or `deep-merge` the member

Reply:
I’m not sure what you mean. Do you want :application/members done differently? Is the new version of :application/invitation-tokens okay? A code example would be helpful.